components/blocks/ActionButton/ActionsButton.tsx

```diff
@@ -26,15 +26,15 @@ export const Actions = ({
           .filter(Boolean)
           .join(' ')}
       >
-        {items?.map((item) => {
+        {items?.map((item, index) => {
           const { variant, label, icon, url, shape } = item;
           {
             const externalUrlPattern = /^((http|https|ftp):\/\/)/;
             const external = externalUrlPattern.test(url);
             const link = url || '#';
             return (
               <ActionButton
-                key={label}
+                key={sanitizeLabel(label) || `action-button-${index}`}
```

Comment:
just use 1 type of key

```suggestion
                key={`action-button-${index}`}
```